components.json

```diff
@@ -757,6 +757,11 @@
 			"title": "R",
 			"owner": "Golmote"
 		},
+		"turtle": {
+			"title": "RDF Turtle/TriG",
```

Comment:
Maybe we should give this a different title? I was searching for this and could find it under T. Maybe just "Turtle" like Wikipedia did.

Regarding `trig`. It would be better to use an alias title. See `markup`.

Reply:
I have changed the title to "Turtle" and added "TriG" as an aliasTitle for "trig".